app/Http/Controllers/UserAuth.php

```diff
@@ -2,14 +2,99 @@
 
 namespace App\Http\Controllers;
 
+use App\Services\Paiments\PaimentsService;
+use App\Services\User\UserService;
+use Session;
+use App\Models\User;
 use Illuminate\Http\Request;
 
 class UserAuth extends Controller
 {
+    public function __construct(protected UserService $userService, protected PaimentsService $paimentsService)
+    {
+    }
     function userLogin(Request $request)
     {
-        Session::put('user', $request->input('user'));
+        $this->userService->addCurrentSessionUser($request->input('user'));
 
         return view('operations');
     }
+
+    public function verification(Request $request){
+//        $name = User::all();
+//        foreach ($name as $value){
+//            if($request['user'] == $value['name'] or $request['mail'] == $value['email']){
+//                echo "identifiant déjà pris";
+//                return view('welcome' );
+//            }
+        //};
+        if ($request->input('password') == null){
```

Comment:
ça peut être intéressant de profiter du système de validation fournir par Laravel :

https://laravel.com/docs/9.x/validation#quick-writing-the-validation-logic

Comme ça, t'auras plus besoin des vérification de genre `if ($request->input('user') == null) {...` et en plus, ça peut être Laravel qui fournit les erreurs de validation :

https://laravel.com/docs/9.x/validation#quick-displaying-the-validation-errors